chromiumcontent/chromiumcontent.gyp

```diff
@@ -39,6 +39,8 @@
         '<(DEPTH)/ppapi/ppapi_internal.gyp:ppapi_proxy',
         '<(DEPTH)/ppapi/ppapi_internal.gyp:ppapi_ipc',
         '<(DEPTH)/ppapi/ppapi_internal.gyp:ppapi_shared',
+        '<(DEPTH)/third_party/libyuv/libyuv.gyp:libyuv',
```

Comment:
`libyuv` is already depended by many components, I don't think we need to add it explicitly.